koin-projects/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.0.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-rc-1-bin.zip
```

Comment:
we need to be careful here, as it could break the build script (jcenter one)

I need to improve CI to detect that install build is broken :/